packages/lib-runtime/tsconfig.json

```diff
@@ -2,8 +2,7 @@
   "extends": "@monorepo/common/tsconfig-bases/react-esm.json",
   "compilerOptions": {
     "outDir": "dist",
-    "declarationDir": "types",
-    "emitDeclarationOnly": true
```

Comment:
`@rollup/plugin-typescript` ignores the `emitDeclarationOnly` option: https://github.com/rollup/plugins/tree/master/packages/typescript/#ignored-options